bolt-modules/boltlib/spec/functions/resolve_references_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/plugin'
+
+describe 'resolve_references' do
+  include PuppetlabsSpec::Fixtures
+  let(:boltdir) { Bolt::Boltdir.new('./spec/fixtures') }
+  let(:config) { Bolt::Config.new(boltdir, {}) }
+  let(:pal) { Bolt::PAL.new(config.modulepath, config.hiera_config, config.boltdir.resource_types) }
+  let(:pdb_client) { Bolt::PuppetDB::Client.new({}) }
+  let(:analytics) { Bolt::Analytics::NoopClient.new }
+  let(:plugins) { Bolt::Plugin.setup(config, pal, pdb_client, analytics) }
+  let(:inventory) { Bolt::Inventory.new({}, plugins: plugins) }
+  let(:tasks_enabled) { true }
```

Comment:
I'm not particularly happy with doing all this setup, but I'm not sure how else to get around it... I tried setting it up similar to (`run_plan_spec`)[https://github.com/puppetlabs/bolt/blob/master/bolt-modules/boltlib/spec/functions/run_plan_spec.rb#L117-L120] but was getting errors about not being able to locate the `Task` plugin or not being able to locate the task `test::references` in `fixtures`.